src/main/java/org/fundacionjala/coding/jimmy/Kata1.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.jimmy;
+
+/**
+ * This class contains a method which is going to spin a string whether is greater than 5 in length.
+ */
+public class Kata1 {
+
+    public static final int THRESHOLD = 5;
+
+    /**
+     * @param wordsToSpin is a method for spin a word greater than 5 in length.
+     * @return a word spinned if is greater than 5 in length.
+     */
+    public String spinWords(String wordsToSpin) {
+        String[] arrayWords = wordsToSpin.split(" ");
+        String string; String space = " ";
+        StringBuffer buffer = new StringBuffer();
```

Comment:
@AT-06/at-06 
Instead of using a StringBuffer
you can use a StringJoiner
https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html

Advantanges
you will not need to add spaces in each iteration and trim the result.